Sprint #23 try#1

nim0y opened the pull request.

nim0y committed:
Sprint #23 try#1

IliaAnK commented on app/src/main/res/layout/fragment_current_playlist.xml:
```diff
@@ -0,0 +1,247 @@
+        <TextView
+            android:id="@+id/share_menu_current_playlist"
+            style="@style/origin"
+            android:layout_width="wrap_content"
```
Рекомендация:
Будет лучше сделать `match_parent` и перевести каждый элемент меню на симметричные вертикальные `padding`-и вместо одно `marginTop`. Таки образом область для клика станет значительно больше и пользователю будет удобнее нажимать на элемент.
Сейчас область очень маленькая, нужно прям в текст попасть.

Аналогично `edit_menu_current_playlist` и `delete_menu_current_playlist `

nim0y committed:
Sprint #23 try#2
Постарался учесть твои замечания,
Не очень понял с цветом курсора, не нашел скрин с тем каким он должен быть.
Надеюсь с меню правильно сделал, то же не очень мне понятно ()

IliaAnK commented:
Для примера:
<img width="409" alt="Снимок экрана 2024-03-06 в 9 54 01 AM" src="https://github.com/nim0y/PlaylistMaker/assets/135727674/d9805a27-6e25-4311-8023-7d9e005e9958">

nim0y committed:
Sprint #23 try#3
